cunumeric/module.py

```diff
@@ -6505,10 +6499,7 @@ def is_diff(arr1, arr2):
 def reshuffle_reshape(arr, axes_set):
     ndim = len(arr.shape)
 
-    if not is_sorted(axes_set):
-        sorted_axes = tuple(sort(axes_set))
-    else:
-        sorted_axes = tuple(axes_set)
+    sorted_axes = tuple(sort(axes_set))
```

Comment:
use `sorted`, which is the python built-in function, not `sort`, which here points to `cunumeric.sort`

Reply:
Done.